docs/recipes/integration-with-sentry.md

```diff
@@ -0,0 +1,119 @@
+# Integration with Sentry
+
+If you are using Sentry as your Error Tracker, you can use this recipe to trace and log actions as *breadcrumbs*.
+
+First you'll need to initialize Sentry in your app using `Sentry.init(...)` in the `main.ts` file.
+
+```ts
+Sentry.init({
+  dsn: ...
+  // ...
+});
+``` 
+
+Next, you need to provide `Sentry`'s `ErrorHandler` and `TraceService`.
+
+```ts
+// app.module.ts
+@NgModule({
+  providers: [
+    // ...
+    {
+      provide: ErrorHandler,
+      useValue: Sentry.createErrorHandler(),
+    },
+    {
+      provide: Sentry.TraceService,
+      deps: [Router],
+    },
+    {
+      provide: APP_INITIALIZER,
+      useFactory: () => () => {},
+      deps: [Sentry.TraceService],
+      multi: true,
+    },
+  ],  
+})
+export class AppModule {}
+
+//app.config.ts
+export const appConfig: ApplicationConfig = {
+  providers: [
+    // ...
+    {
+      provide: ErrorHandler,
+      useValue: Sentry.createErrorHandler(),
+    },
+    {
+      provide: Sentry.TraceService,
+      deps: [Router],
+    },
+    {
+      provide: APP_INITIALIZER,
```

Comment:
do you think we can use the `provideAppInitializer` which is part of angular 19?
https://angular.dev/api/core/provideAppInitializer?tab=api